Source/Blazorise/Base/BaseComponent.cs

```diff
@@ -711,6 +716,21 @@ public TextOverflow TextOverflow
         }
     }
 
+    /// <summary>
+    /// Determines the font size of an element.
+    /// </summary>
+    [Parameter]
+    public TextSize TextSize
+    {
+        get => textSize;
+        set
+        {
+            textSize = value;

```

Comment:
I think i've seen you use the approach of checking whether the TextSize is already the value in most places? any reason why we're not doing that here?

Reply:
We need to revisit that. In the past with older Blazor, there were problems in some instances, and checking for the value change would not work. The component would not refresh when needed. I don't remember the reason, but I know that I had to be careful where to put the extra condition.

Reply:
Closing for now. I will open a new ticket for these optimizations.

Reply:
It would be good to move it to the OnParametersAsync, and try to develop a more concise api or helpers for these kind of stuff. 
I.E `DirtyIfParameterChanged(x=> x.MyParameter); `